troubleshooting.md

```diff
@@ -0,0 +1,211 @@
+# Troubleshooting guide
+
+This guide will help you sort out the most common issues of running ITS on the PiDP-10.
+Issues covered so far:
+
+* [COMSAT is crashing on start](#comsat-is-crashing-on-start)
+
+through out the document you will see certain special characters mentioned. They are slightly different depending on your keyboard and terminal
+* `$` means the ALT or ESC key
+* `^` means the CTRL or STRG key
+* `<escape>` means the ALT or ESC key in EMACS
+* `<control>` means the Control key in EMACS
+
+## COMSAT is crashing on start
+Thanks to eswenson for the intial steps here
+
+In order for INQUIR entries to stick, you must have COMSAT running. 
+
+If you run PEEK, you should see two COMSAT jobs. One has the JNAME IV and the other JOB.nn.  If these jobs are not present, then COMSAT may have started and died or not started at all
+```
+*:peek
+KA ITS 1651  Peek 631   8/14/2024 09:59:35  Up time = 5:05
+Memory: Free=457   Runnable Total=11 Out=3     Users: High=13 Runnable=0
+Index Uname Jname Sname     Status   TTY    Core Out %Time    Time PIs
+  0 SYS    SYS    SYS        HANG    ?        71   0   0%          1
+  1 CORE   JOB    CORE       UUO     ?         0   0   0%
+  2 MIKEK  HACTRN MIKEK      HANG    >        30   9   0%
+ 12  MIKEK  PEEK   MIKEK     +TTYBO  T52 C    11   2   0%
+  3 .BATCH BATCHN .BATCH     SLEEP   ?       126  23   0%
+  4 TARAKA CNAVRL CNAVRL     10!0    ? DSN    29   0   0%          .VALUE
+  5 GUNNER GUNNER GUNNER    _SLEEP   ?        11   3   0%
+  6 TARAKA PAPSAV PAPSAV     HANG    ?         1   0   0%
+  7 TARAKA NAMDRG NAMDRG     HANG    ?        29   0   0%
+ 10 PFTHMG DRAGON PFTHMG     HANG    ?         6   0   0%
+ 11 TARAKA JOB.07 SYS        HANG    ?         3   0   0%
+Fair Share 99%     Totals:                   317       0%          1
+Logout time =         Lost 0%  Idle 98%  Null time = 5:07
+```
+As you can see above none of the COMSAT processes are running.
+
+There are several reasons why COMSAT may die upon startup The most common are:
+Lets start going through those one by one:
+
+### Network parameters for COMSAT are not correct.
+When you bring up KA ITS, you'll see a message on the operator console like this:
+
+ LOGIN  TARAKA 0 12:09:11
+TOP LEVEL INTERRUPT 200 DETACHED JOB # 4, USR:COMSAT IV     12:09:12
+
+This means that COMSAT has crashed.
+
+If you look at the IP address that COMSAT is configured with:
+```
+comsat$j
+$l .mail.;comsat launch
+bughst/'NEW$:   SHOWQ+50,,PAT+6   =30052000544
+```
+
+you'll note that that octal address is: 192.168.1.100
+
```

Comment:
I recommend you insert my writeup on how to decode those octal IP addresses and convert them to standard octet notation. See recent post to PiDP10 mailing list.